src/OnboardingSPA/steps/SiteGen/Editor/Header/stylesheet.scss

```diff
@@ -160,3 +160,26 @@
 		}
 	}
 }
+
+.nfd-interface-interface-skeleton__editor {
+
+	&.editor-actions-disabled {
+
+		.nfd-editor-actions-overlay {
```

Comment:
Can we also add `cursor: not-allowed;` so that it's clear that no interaction is allowed?